torchci/lib/bot/mergeBot.ts

```diff
@@ -59,7 +61,7 @@ function mergeBot(app: Probot): void {
       await dispatchEvent("try-revert");
       await reactOnComment(ctx, "+1");
     }
-    else if (commentBody.match(rebaseCmdPat) && ctx.payload.comment.user.login == "clee2000") {
+    else if (commentBody.match(rebaseCmdPat) && rebaseAllowList.includes(ctx.payload.comment.user.login)) {
```

Comment:
I just realized you were blocking this functionality based on the commenter, but would it maybe make more sense to block on the PR author? The reason being this seems like a super power where we can go ruin other people's PRs, which doesn't sound great.

Reply:
Or both! Like commenter has to be a part of some group AND PR author needs to agree to have their stuff rebased.

Reply:
Changed it so that PR author and PR commenter have to be on the list, although hopefully the people on this list aren't going to be going around ruining random people's PRs

In the future, perhaps we should consider maintaining that the commenter has to be on some kind of allow list? Otherwise anyone can just rebase anywhere.

Reply:
I thought the whole point was to be able to rebase outsider's PRs?

Reply:
I'm still gatekeeping it for now which is why theres a list for both PR author and for commenter

Regarding the future, maybe we should always check the commenter?  If we just check commenter, then we can still rebase external PRs